airflow/ui/src/components/FlexibleForm/NormalRow.tsx

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Field, Stack } from "@chakra-ui/react";
+import Markdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+import type { ParamSpec } from "src/queries/useDagParams";
+
+import type { FlexibleFormElementProps } from ".";
+import { SelectElement } from "./SelectElement";
+
+const isRequired = (param: ParamSpec) =>
+  // The field is required if the schema type is defined.
+  // But if the type "null" is included, then the field is not required.
+  // We assume that "null" is only defined if the type is an array.
+  Boolean(param.schema.type) && (!Array.isArray(param.schema.type) || !param.schema.type.includes("null"));
+
+/** Render a normal form row with a field that is auto-selected */
+export const NormalRow = ({ key, name, param }: FlexibleFormElementProps) => (
+  <Field.Root orientation="horizontal" required={isRequired(param)}>
+    <Stack css={{ "flex-basis": "30%" }}>
+      <Field.Label css={{ "flex-basis": "0" }} fontSize="md">
+        {param.schema.title ?? name} <Field.RequiredIndicator />
+      </Field.Label>
+    </Stack>
+    <Stack css={{ "flex-basis": "70%" }}>
```

Comment:
```suggestion
    <Stack flexBasis="30%">
      <Field.Label flexBasis="0" fontSize="md">
        {param.schema.title ?? name} <Field.RequiredIndicator />
      </Field.Label>
    </Stack>
    <Stack flexBasis="70%">
```

Reply:
I attempted the adjust the `flex-basis`as proposed but this destroyed the layout... do you have an idea why? Just does NOT parse the flex setting to the element when I inspect the outcome rendered :-(

Reply:
Actually we only need the `flexBasis="70%"` all the others can be removed. but what are you seeing when you do that?

Reply:
Yeah, one point is right - the definition with 70% is sufficient!

But if I change from `<Stack css={{ "flex-basis": "70%" }}>` to `<Stack flex-basis="70%">`

The form switches to
![image](https://github.com/user-attachments/assets/881acd33-490a-41e2-868a-1b9f89b5dacb)

Reverting back to the ugly/complex/indirect CSS statement it makes it clean/pretty. Maybe I am just too stupid but with your proposed approach the inspector does not show the `flex-basis` at the generated `<div>` at all :-O 
![image](https://github.com/user-attachments/assets/16843024-d1ce-4603-8df7-a12d227c0878)

Reply:
Ok let's leave it as-sis then